### DISCUSSION TITLE
Gaps and margins

### DISCUSSION THREAD

**Discussion by apastuszak** [ianyh/Amethyst#1535] [Q&A]
I have my dock set to auto-hide.

I am trying to configure Amethyst to show the gap between windows to be the same as the screen padding.

This is my setting:
![image](https://github.com/ianyh/Amethyst/assets/3401054/254fbf13-d206-4ed8-8e20-a11a8d947825)

When I do this, the gap between windows is smaller than the screen padding value.

**Comment by ianyh** [comment]
If I recall correctly, window margins are on all sides, so the outer margins are added to the screen padding.
